### DISCUSSION TITLE
PatchTool 6.2 9/3

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#228] [Announcements]
修正部分翻譯，應對維護更新。

檔案：
- FFXIVChnTextPatch-0903.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-0903.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-0903.rar：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.69'>PatchTool 6.2 9/3</a>.</em>

**Comment by sdadasx123** [comment]
想請問目前這個是6.2的劇情還沒漢化嗎？
我只是想確定一下不是我用錯，沒有要催漢化的意思

**Reply by GpointChen** [reply]
是的

**Comment by eddie1231306** [comment]
想請問下，我6.1版本的時候暫時退坑了，直到剛剛想回來玩，然後我就直接更新了，直到出現下圖
![unknown](https://user-images.githubusercontent.com/113383455/189732141-31edbe39-f375-4e7f-8769-106ead540e84.png)
我詢問了我身邊的朋友，他們猜測可能是我忘記還原漢化所導致的，我已經把漢化的程式、遊戲主程式都刪除掉了，現在還是出現這個問題，不知道是不是有漢化的部分沒有刪除乾淨，或是版主有遇過類似的問題可以幫我解決?

**Reply by eddie1231306** [reply]
不知道有沒有除了重灌以外的方式可以解決掉...

**Reply by GpointChen** [reply]
漢化程式是去覆蓋你原本的`\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`資料夾內的`0a0000.win32.dat0`等六個檔案，單純的刪除漢化程式和遊戲主程式都不會恢復。

你可能可以（擇一）
1. 尋找當初漢化程式`backup`資料夾內的備份檔然後手動覆蓋
2. 在過去的release中也有許多乾淨的backup，如果你記得自己最後是在哪個版本暫離可以找看看相對應的還原檔
3. 驗證遊戲完整性（不過這也很花時間）

**Comment by Bos-hunting** [comment]
請問有漢化了嗎，今天國服好像有6.15的更新了:)
